src/blocks/go-search/index.js

```diff
@@ -0,0 +1,76 @@
+/* global coblocksBlockData */
+
+/**
+ * Internal dependencies
+ */
+import edit from './edit';
+import metadata from './block.json';
+
+/**
+ * WordPress dependencies
+ */
+import { __, _x } from '@wordpress/i18n';
+import { Dashicon } from '@wordpress/components';
+
+/**
+ * Block constants
+ */
+const { name, category, attributes } = metadata;
+
+const settings = {
+	attributes,
+	/* translators: block description */
+	description: __( 'Go search field.', 'coblocks' ),
+	edit,
+	example: {
+		attributes: {
+			count: 2,
+		},
```

Comment:
It does not look like we use this attribute. Is the idea to have a render of multiple blocks on the same page?

Reply:
@AnthonyLedesma That's a good question. Honestly, these styles were just carried over from Go with some minor adjustments, so I'm not entirely sure what this lines intention is.